subsys/ipc/ipc_service/backends/ipc_icmsg_me_follower.c

```diff
@@ -0,0 +1,375 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+
+#include <zephyr/device.h>
+#include <zephyr/ipc/icmsg.h>
+#include <zephyr/ipc/ipc_service_backend.h>
+
+#define DT_DRV_COMPAT	zephyr_ipc_icmsg_me_follower
+
+#define INVALID_EPT_ID 255
+#define SEND_BUF_SIZE CONFIG_IPC_SERVICE_BACKEND_ICMSG_ME_SEND_BUF_SIZE
+#define NUM_EP        CONFIG_IPC_SERVICE_BACKEND_ICMSG_ME_NUM_EP
```

Comment:
Would it make sense to define the number of endpoints as an attribute of the IPC node in DTS? There might be multiple IPC nodes in the system, and not all of them may need the same number of available endpoints.
The `SEND_BUF_SIZE` could also be considered as node attribute in DTS.

Reply:
It would be challenging to implement. I'll think how this could be done :)

I hope it could be introduced as an additional feature later?

Reply:
It could be something equivalent to: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/spi/spi_nrfx_spim.c#L570
Requires more macro magic, but is not too complicated.

It can be introduced later.